torch/_functorch/_aot_autograd/dispatch_and_compile_graph.py

```diff
@@ -102,6 +103,14 @@ def aot_dispatch_base_graph(
     copy_count2 = assert_functional_graph(fw_module.graph)
     propagate_input_mutation_stacktraces(fw_module.graph)
 
+    # See Note [Side-Effectful Tokens in AOTAutograd]
+    num_tokens = len(fw_metadata.tokens)
+    if num_tokens != 0 and aot_config.backend_name == "inductor":
```

Comment:
Is it possible to add this as a config option instead to AOTAutograd (e.g. torch._functorch.config)? Inductor is probably not the only backend that will want this